onnx/backend/test/case/node/svd.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Any
+
+import numpy as np  # type: ignore
+
+import onnx
+
+from ..base import Base
+from . import expect
+
+
+class Svd(Base):
+    @staticmethod
+    def export() -> None:
+        dimensions = [((3, 4), "2d"), ((2, 3, 4), "3d"), ((5, 4, 3, 2, 1, 3, 4), "7d")]
+
+        for compute_uv in [0, 1, None]:
+            for full_matrices in [0, 1, None]:
+                for (dims, dims_name) in dimensions:
+                    A = np.random.randn(*dims)
+
+                    node_args: Any = {"inputs": ["A"]}
+                    numpy_args = {}
+                    name = f"test_svd_{dims_name}"
+
+                    if full_matrices is not None:
+                        node_args["full_matrices"] = full_matrices
+                        numpy_args["full_matrices"] = bool(full_matrices)
```

Comment:
Does it work without bool? In that case, the same dictionary could be used to hold parameters for the node and the numpy function.

Reply:
Good catch! Added